src/panels/SkyBrowserPanel/BrowserTabs/Settings.tsx

```diff
@@ -1,54 +1,193 @@
+  useBrowserFov,
+  useBrowserRadius
+} from '../hooks';
+
+import { SettingsDisplayCopies } from './SettingsDisplayCopies';
 
 interface Props {
-  id: string | undefined;
+  id: string;
 }
 
 export function Settings({ id }: Props) {
   const luaApi = useOpenSpaceApi();
-  const selectedBrowserId = useAppSelector((state) => state.skybrowser.selectedBrowserId);
-  const color = useBrowserColorString(selectedBrowserId);
-  const browserIds = useSkyBrowserIds();
+  const targetId = useAppSelector((state) => state.skybrowser.browsers[id]?.targetId);
+  const color = useBrowserColorString(id);
+  const fov = useBrowserFov(id);
+  const radius = useBrowserRadius(id);
+  const { ra, dec } = useBrowserCoords(id);
 
   function setColor(newColor: string) {
     const parsedColor = newColor.match(/\d+/g)?.map(Number);
     if (!parsedColor) {
       return;
     }
     const [r, g, b] = parsedColor;
-    luaApi?.skybrowser.setBorderColor(selectedBrowserId, r, g, b);
+    luaApi?.skybrowser.setBorderColor(id, r, g, b);
   }
 
-  function deleteBrowser() {
-    if (!id) {
-      return;
-    }
-    // If there are more browsers, select another browser
-    if (browserIds.length > 1) {
-      const otherBrowsers = browserIds.filter((b) => b !== id);
-      luaApi?.skybrowser.setSelectedBrowser(otherBrowsers[0]);
-    }
-    luaApi?.skybrowser.removeTargetBrowserPair(id);
+  function setBorderRadius(newValue: number): void {
+    luaApi?.skybrowser.setBorderRadius(id, newValue);
+  }
+
+  function setVerticalFov(newValue: number): void {
+    luaApi?.skybrowser.setVerticalFov(id, newValue);
+  }
+
+  function setRightAscension(newValue: number): void {
+    luaApi?.skybrowser.setEquatorialAim(id, newValue, dec);
+  }
+
+  function setDeclination(newValue: number): void {
+    luaApi?.skybrowser.setEquatorialAim(id, ra, newValue);
   }
 
   return (
-    <Stack gap={5} my={'lg'}>
+    <Box py={'xs'}>
       <Title order={2}>Settings</Title>
-      <ColorInput
-        label={'Color'}
-        placeholder={'Set browser color...'}
-        format={'rgb'}
-        defaultValue={color}
-        onChange={setColor}
-      />
-      <Button variant={'outline'} color={'red'} onClick={deleteBrowser} mt={'lg'}>
-        Delete browser
-      </Button>
-    </Stack>
+      <Tabs defaultValue={'Settings'}>
+        <Tabs.List>
+          <Tooltip label={'Settings specific for this browser.'}>
+            <Tabs.Tab value={'Settings'}>This Browser</Tabs.Tab>
+          </Tooltip>
+          <Tooltip
+            label={`Settings for Display Copies, which are a way to render the WorldWide
+                Telescope view as a ScreenSpaceRenderable.`}
+          >
+            <Tabs.Tab value={'Display'}>Display Copies</Tabs.Tab>
+          </Tooltip>
+          <Tooltip label={`General settings for all browsers.`}>
+            <Tabs.Tab value={'General'}>General</Tabs.Tab>
+          </Tooltip>
+        </Tabs.List>
+        <Tabs.Panel value={'Settings'} py={'sm'}>
```

Comment:
Now you add a `py={'sm'}` to all panels individually. This easily becomes out of sync

How about, instead, wrapping all the `Tabs.Panel` components in a box or something that adds that padding to all of them?